index.html

```diff
@@ -227,10 +227,8 @@
 };
 
 interface IdleDeadline {
-  readonly attribute DOMHighResTimeStamp deadline;
+  double timeRemaining();
```

Comment:
I think you may be looking at an outdated definition.

"The DOMHighResTimeStamp type is used to store a time value measured relative from the time origin *or a time value that represents a duration between two DOMHighResTimeStamps*." - [source](http://w3c.github.io/hr-time/#dom-domhighrestimestamp)